robotbase/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,62 +4,48 @@
 
 package frc.robot;
 
+import static edu.wpi.first.units.Units.MetersPerSecond;
```

Comment:
Robot container does not provide much use to us. Delete it, and move everything to either Robot.java or other classes